cf-usb-plugin/config-production.yaml

```diff
@@ -0,0 +1,21 @@
+# Production configuration for the cf-usb-plugin pipelines
+# Note that the secrets are not here; they are in $CONCOURSE_SECRETS_FILE
+
+# should be upstream: cloudfoundry/cli-plugin-repo
+upstream: SUSE/cli-plugin-repo
```

Comment:
Upstream for the production config is not SUSE.

Reply:
Fixed in 6a50579